redisinsight/api/src/utils/cli-helper.ts

```diff
@@ -264,6 +264,11 @@ export const getASCIISafeStringFromBuffer = (reply: Buffer): string => {
   return result;
 };
 
+export const getUTG8FromBuffer = (reply: Buffer): string => {
+  const decoder = new TextDecoder('utf-8');
+  return decoder.decode(new Uint8Array(reply));
+};
+
```

Comment:
Is there a reason to use TextDecoder and Uint8Array here in node?
Could you please check api/src/common folder. we have transformers there. It will be better to reuse them

Reply:
TextDecoder still use ascii formatter...

Reply:
and in api/src/common/ functions which working with strings, but we get buffer in response

Reply:
Ok. let's not use transformer from common for now (I'm not happy with this) BUT let's implement conversion in a right way then. We don't need double conversion here. you can simpy do `reply.toString('utf8')`